.github/workflows/build.yml

```diff
@@ -7,16 +7,48 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  generate_wheels_matrix:
+    name: Generate wheels matrix
+    runs-on: ubuntu-latest
+    outputs:
+      include: ${{ steps.set-matrix.outputs.include }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Checkout mypy
```

Comment:
possibly dumb question: what do we need the mypy checkout here for? (we need the mypy_mypyc-wheels checkout for cibuildwheel.toml)